pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[tool.poetry]
+name = "netbox-loadtest"
+version = "2.0.1"
+description = "A load test script for the netbox IPAM solution."
+authors = ["Jarrod J Manzer <jjmanzer@godaddy.com>"]
+license = "MIT"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Environment :: Web Environment",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+]
+packages = [
+  { include = "bin/*.py" },
+]
+
+[[tool.poetry.source]]
+name = "GitHub"
+url = "https://github.com/jjmanzer/netbox-loadtest/"
```

Comment:
This section is unnecessary; in fact, I would say it should not be included. This would only be used if you were trying to use a private PyPi instance, as indicated here: https://python-poetry.org/docs/pyproject/#dependencies-and-dev-dependencies

I think what you might be looking for is the [urls section](https://python-poetry.org/docs/pyproject/#urls):
```toml
[tool.poetry.urls]
"GitHub" = "https://github.com/jjmanzer/netbox-loadtest/"
```
You may also want to add this as a [repository](https://python-poetry.org/docs/pyproject/#repository) link in the main `[tool.poetry]` section.